StripePaymentsUI/StripePaymentsUI/Source/Internal/UI/Elements/DropDownFieldElement+CardBrand.swift

```diff
@@ -14,14 +14,15 @@ import UIKit
 extension DropdownFieldElement {
 
     @_spi(STP) public static func makeCardBrandDropdown(cardBrands: Set<STPCardBrand> = Set<STPCardBrand>(),
+                                                        disallowedCardBrands: Set<STPCardBrand> = Set<STPCardBrand>(),
```

Comment:
Is there a strong reason to have a default?  Seems like something each caller should be required to pass in.

Reply:
We do have the "cardBrands" parameter with a default value too, inside of CardSectionElement we initialize the drop down before we have fetched brands if the merchant is eligible for CBC, so it's nice for convince.